src/js/components/ChatControl/ChatControl.js

```diff
@@ -7,17 +7,32 @@ import NoChat from '../NoChat/NoChat';
 const ChatControl = (props) => {
   const context = useContext(GlobalContext);
   const { match } = props;
-  const { Config } = context;
+  const { Config, createRandomMessageLog } = context;
```

Comment:
Is there a way to do this without the random code concerns being in this otherwise production-usable component?